src/mmw/js/src/compare/controllers.js

```diff
@@ -16,28 +18,87 @@ var CompareController = {
             App.currProject
                 .fetch()
                 .done(function() {
+                    setupProjectCopy();
```

Comment:
Instead of copying the project, is it possible for us to use and/or expand some of the logic we are using for public/read-only projects? Copying the project seems a bit heavy-handed to me. I think it would be great if we had some sort of read-only switch that we could put the project in, where changes to the project wouldn't trigger a save. 

I'm curious what others think.